packages/url_launcher/url_launcher/lib/url_launcher.dart

```diff
@@ -62,7 +62,7 @@ import 'package:url_launcher_platform_interface/url_launcher_platform_interface.
 /// is set to true and the universal link failed to launch.
 Future<bool> launch(
   String urlString, {
-  bool forceSafariVC = true,
+  bool forceSafariVC = false,
```

Comment:
any chance we can add a test? It could be as simple as just calling `launch('<some-url', forceWebView: true)` and asserting that it doesn't throw.

Reply:
@blasten
I've commited the tests.
Also found out that the test 'default behavior' needed an update, because it expected useSafariVC = true.

Reply:
Also, the example still has sdk constraints which don't enable nnbd, so `flutter analyze` gives the following output:
```
  error • This requires the 'non-nullable' language feature to be enabled •
         example/lib/main.dart:30:18 • experiment_not_enabled
  error • The parameter type 'dynamic' is incompatible with the field type
         'String' • example/lib/main.dart:30:25 •
         field_initializing_formal_not_assignable
  error • required isn't a type • example/lib/main.dart:30:25 • not_a_type
  error • This requires the 'non-nullable' language feature to be enabled •
         example/lib/main.dart:38:15 • experiment_not_enabled
```

Btw. `analysis_options.yaml` has:
```
analyzer:
  enable-experiment:
    - non-nullable
```
which is not necessary any more.

Should I also change both things?